instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/InjectionState.java

```diff
@@ -20,11 +20,6 @@ public InjectionState(SnippetInjectingResponseWrapper wrapper) {
     this.characterEncoding = wrapper.getCharacterEncoding();
   }
 
-  public InjectionState(String characterEncoding) {
```

Comment:
remove `@Nullable` from wrapper field above

and remove characterEncoding field